Integrations/integration-CounterCraft.yml

```diff
@@ -0,0 +1,1046 @@
+    description: Create a new deception campaign
+  - name: countercraft-list-dsns
+    arguments: []
+    outputs:
+    - contextPath: CounterCraft.DSN.Id
+      description: ID
+      type: number
+    - contextPath: CounterCraft.DSN.Name
+      description: Name
+      type: string
+    - contextPath: CounterCraft.DSN.Description
+      description: Description
+      type: string
+    - contextPath: CounterCraft.DSN.Hostname
+      description: Hostname
+      type: string
+    - contextPath: CounterCraft.DSN.Port
+      description: Port
+      type: number
+    description: List Deception Support Nodes (DSNs)
+  - name: countercraft-list-providers
+    arguments: []
+    outputs:
+    - contextPath: CounterCraft.Provider.Id
+      description: ID
+      type: number
+    - contextPath: CounterCraft.Provider.Name
+      description: Name
+      type: string
+    - contextPath: CounterCraft.Provider.Description
+      description: Description
+      type: string
+    - contextPath: CounterCraft.Provider.Type_Code
+      description: Type
+      type: string
+    - contextPath: CounterCraft.Provider.Status_Code
+      description: Status
+      type: string
+    description: 'List providers (providers for hosts or services i.e. AWS or Office365) '
+  - name: countercraft-create-host-machine
+    arguments:
+    - name: name
+      required: true
+      description: Name
+    - name: description
+      description: Description
+    - name: provider_id
+      required: true
+      description: Provider
+    - name: deception_support_node_id
+      required: true
+      description: Deception Support Node ID
+    - name: campaign_id
+      required: true
+      description: Campaign
+    - name: ip_address
+      required: true
+      description: IP Address
+    - name: username
+      required: true
+      description: Username
+    - name: password
+      required: true
+      description: Password
+    outputs:
+    - contextPath: CounterCraft.Host.Id
+      description: Host ID
+      type: number
+    description: Deploy a new deception host
+  - name: countercraft-list-incidents
+    arguments:
+    - name: campaign_id
+      required: true
+      description: Campaign ID
+    outputs:
+    - contextPath: CounterCraft.Incident.Id
+      description: Incident ID
+      type: number
+    - contextPath: CounterCraft.Incident.Name
+      description: Name
+      type: string
+    - contextPath: CounterCraft.Incident.Description
+      description: Description
+      type: string
+    - contextPath: CounterCraft.Incident.Status_Code
+      description: Status
+      type: string
+    - contextPath: CounterCraft.Incident.TPL_Code
+      description: TLP
+      type: string
+    description: List all incidents currently active
+  - name: countercraft-manage-campaign
+    arguments:
+    - name: campaign_id
+      required: true
+      description: Campaign ID
+    - name: operation
+      required: true
+      description: Operation
```

Comment:
add outputs (if not to context - at least something that the user will know with the operation succeeded).